bin/concat_tsv.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+#
+# Author: Sabrina Krakau <sabrina.krakau@uni-tuebingen.de>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+####################################################################################################
+
+import argparse
+import sys
+import os
+
+import pandas as pd
+
+####################################################################################################
+
+
+def parse_args(args=None):
+    """Parses the command line arguments specified by the user."""
+    parser = argparse.ArgumentParser(description="Concatenate TSV files.")
+
+    # INPUT FILES
+    parser.add_argument("-i", "--input", help="Path to input files.", type=str, required=True, nargs="+")
+
+    # OUTPUT FILES
+    parser.add_argument("-o", "--output", help="Path to output file.", type=str, required=True)
+
+    # PARAMETERS
+    parser.add_argument(
+        "-c",
+        "--chunk-size",
+        help="Chunk size used to process TSV files. Default: 10000",
```

Comment:
This may be a bit random, but for future debugging it could help to specify that the chunks are defined as number of lines (not e.g. number of files)